service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/StatementOfServiceController.java

```diff
@@ -27,17 +27,17 @@
     private final UserDetailsService userDetailsService;
     private final MapperService mapperService;
     private final StatementOfServiceService statementOfServiceService;
+    private final RequestData requestData;
 
     @PostMapping("/about-to-start")
     public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
-        @RequestHeader(value = "authorization") String authorization,
         @RequestBody CallbackRequest callbackRequest) {
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         CaseData caseData = mapperService.mapObject(caseDetails.getData(), CaseData.class);
 
         caseDetails.getData().put("statementOfService", statementOfServiceService.expandRecipientCollection(caseData));
 
-        String label = String.format(CONSENT_TEMPLATE, userDetailsService.getUserName(authorization));
+        String label = String.format(CONSENT_TEMPLATE, userDetailsService.getUserName(requestData.authorisation()));
```

Comment:
RequestData not needed here - it can be injected to userDetailsService and authorisation removed from the parameters.